component/datamodel/util/write.js

```diff
@@ -0,0 +1,33 @@
+// Copyright IBM Corp. 2015,2016. All Rights Reserved.
+// Node module: loopback-workspace
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+'use strict';
+const fs = require('fs-extra');
+const path = require('path');
+
+class WriteOperations {
```

Comment:
instead of having functional code we are having a class with all write operations